.github/filters/ci.yml

```diff
@@ -73,7 +71,7 @@ rust-changes: &rust-changes
 # > Note: We don't use `rust-changes` because it include all the change made to the rust code
 # > including change made to the ionic part.
 python-jobs:
-  - *ci-workflow
+  - .github/workflows/ci-python.yml
```

Comment:
What about the `ci-workflow.yml`'s own dependencies ?
For instance if something in `.github/actions/` changes the build must be triggered

Reply:
given there is not much things in `.github/actions` and it doesn't changes often, better play it safe and add the whole directory instead of just what we need ;-)

Reply:
Should be good now